components/EventsTiles.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import TextSection from './TextSection';
+import Heading from './Heading';
+import { fetchContent } from '../api/apiRoot';
+import Image from 'next/image';
+
+// const events = [
+//   {
+//     id: 1,
+//     name: 'Event Name',
+//     description: 'Our biggest event to date It saw students from all over Canada come up with innovative solutions',
+//   },
+//   {
+//     id: 2,
+//     name: 'Event Name',
+//     description: 'Our biggest event to date It saw students from all over Canada come up with innovative solutions',
+//   },
+//   {
+//     id: 3,
+//     name: 'Event Name',
+//     description:
+//       'Our biggest event to date It saw students from all over Canada come up with innovative solution kfhewiuo akjlsfhakjdlf hsdklf hsadklf hsdakjl hasdk',
+//   },
+// ];
+
+const EVENTS_TILES_CONTAINER = 'w-4/5 mx-auto flex flex-wrap md:flex hidden';
+const EVENT_TILE = 'flex flex-col h-96 bg-white rounded-3xl items-center px-4 justify-evenly';
+const EVENT_SVG = '/svgs/homepage/events.svg';
+
+const EventTile = ({ eventName, description }) => {
+  const descriptionText = description.content[0].content[0].value;
+  return (
+    <div className={EVENT_TILE}>
+      <Heading classes="text-3xl font-medium" underlineType="None">
+        {eventName}
+      </Heading>
+      <Image src={EVENT_SVG} alt="event image" width={180} height={180} objectFit="cover" />
+      <TextSection classes="text-sm">{descriptionText}</TextSection>
+    </div>
+  );
+};
+
+const EventsTiles = () => {
+  const [events, setEvents] = useState([]);
```

Comment:
Instead of using useEffect(), NextJS has a built in function called getStaticProps(). This allows you to prerender content on the site resulting better performance and SEO.

I did something similar in PR #107 

Here is the documentation if you are interested.
https://nextjs.org/docs/pages/building-your-application/data-fetching/get-static-props